### ISSUE TITLE
Feature: Marker / Highlighter

### ISSUE THREAD

**Issue by Haroenv** [excalidraw/excalidraw#1317]
This can almost be done with the line tool, however the stroke is too small, and by default it goes on top of the already existing drawing.

My suggestion would be:

Thickness: text + some margin
Place at background so the text can still clearly be read

**Comment by dwelle** [comment]
It can be done right now with a rectangle shape: 

![image](https://user-images.githubusercontent.com/5153846/78771467-7ac1e380-7990-11ea-93d7-2904ff2eca6b.png)

Though the workflow is not ideal.

Anyway, this might be part of https://github.com/excalidraw/excalidraw/issues/1126

**Comment by Haroenv** [comment]
Yes, found the flow for using rectangle less useful since my other boxes were empty so I had to toggle all the settings back and forth. In the end I went with underlining in my drawing since it was easiest.

**Comment by lipis** [comment]
I would update the other issue and have them tackle at the same time... text color and background fo ranges.

**Comment by senseofeternity** [comment]
Still need this feature, native highlighter could be very useful

**Comment by lucadonnoh** [comment]
please!

**Comment by duke8585** [comment]
please :)

**Comment by VasoTodorovic** [comment]
please

**Comment by RenTensei** [comment]
'+'

**Comment by Pranoy1c** [comment]
Another please! This would be very useful.

**Comment by eevmanu** [comment]
https://github.com/excalidraw/excalidraw/issues/1317#issuecomment-610867922

Step by step on how is possible to do that with rectangle shape.

<img width="640" height="360" alt="Image" src="https://github.com/user-attachments/assets/d8b6310c-3732-41f4-86eb-f8ec883825de" />

pd: hopefully someone could find it useful

**Comment by eevmanu** [comment]
looks like is being tracked on #4323

**Comment by bjornasm** [comment]
Isnt that something else? A highlighter is a very broad stroked pen that is see through by default, imo.
